example-form/components/FormRegisterPetDetails/FormRegisterPetDetailsContainer.tsx

```diff
@@ -16,12 +17,24 @@ export const FormRegisterPetDetailsContainer = ({
 	introduction: string;
 	callToAction?: ReactNode;
 }) => {
-	const { hasCompletedPreviousSteps } = useFormExampleMultiStep();
+	const titleRef = useRef<HTMLHeadingElement>(null);
+	const { hasCompletedPreviousSteps, currentStep } =
+		useFormRegisterPetDetails();
+
+	// Focus the title of the current step as the user navigates between form steps
+	useEffect(() => {
```

Comment:
What we are doing here is focusing the title of the form whenever the uses changes step. Currently it sets the focus to the top of the page which is not great.